test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Rendering/HtmlHelperHiddenTest.cs

```diff
@@ -413,10 +413,9 @@ public void HiddenWithEmptyNameAndPrefixThrows()
             {
                 { "class", "some-class"}
             };
-            var expected = "The name of an HTML field cannot be null or empty. Instead use methods " +
+            var expected = new ArgumentException("The name of an HTML field cannot be null or empty. Instead use methods " +
                 "Microsoft.AspNetCore.Mvc.Rendering.IHtmlHelper.Editor or Microsoft.AspNetCore.Mvc.Rendering." +
-                "IHtmlHelper`1.EditorFor with a non-empty htmlFieldName argument value." +
-                Environment.NewLine + "Parameter name: expression";
+                "IHtmlHelper`1.EditorFor with a non-empty htmlFieldName argument value.", "expression").Message;
```

Comment:
:zero: need to create the `ArgumentException`. Just get rid of the newline and parameter name.